score-test/src/test/java/bio/overture/score/test/util/Assertions.java

```diff
@@ -67,7 +67,7 @@ private static void compareNow(File[] fileArr, HashMap<String, File> map) throws
           String cSum1 = checksum(fileArr[i]);
           String cSum2 = checksum(fComp);
           if (!cSum1.equals(cSum2)) {
-            fail(fileArr[i].getName() + "\t\t" + "different");
+            //            fail(fileArr[i].getName() + "\t\t" + "different");
```

Comment:
If this is a failure test case we should be testing, we can't just remove the fail assertion... It needs to be replaced by something. Can you explain to me whats going on in this file?